README.md

```diff
@@ -338,6 +338,8 @@ then this option is ignored.
 from the cache. By default _req.originalUrl_ is used as the key.
 * [Optional] statusCodeExpires - Useful if you want different status code responses to be cached for different
 durations.
+* [Optional] exposeHeader - Default responses use `x-expeditious-cache` as the header key. Set a new string value to 
```

Comment:
Could I make one picky request? I feel like `exposeHeader` could be more descriptive if called `cacheStatusHeader`.

Reply:
Yeah, no problem